web/src/routes/(user)/people/+page.svelte

```diff
@@ -32,6 +33,14 @@
 
   let showLoadingSpinner = false;
   let toggleVisibility = false;
+  let potentialMergePeople: PersonResponseDto[];
+
+  let showChangeNameModal = false;
+  let showMergeModal = false;
```

Comment:
It seems like this is basically derived from personMerge2 being set and/or potentialMergePeople being bigger than zero. Do we need all three variables? Also, do we need the boolean, since it's only used to hide/show the modal? I think there is room to simplify this and reduce the amount of things that manually need to get set. You could auto-compute merge people based on people + personName, eg.

Reply:
`potentialMergePeople` can be an empty array if there's only one person with the same name as the name entered by the user